ElfUtils/ElfFileTest.cpp

```diff
@@ -156,3 +156,64 @@ TEST(ElfFile, FileDoesNotExist) {
   EXPECT_THAT(absl::AsciiStrToLower(elf_file.error().message()),
               testing::HasSubstr("no such file or directory"));
 }
+
+TEST(ElfFile, HasDebugInfo) {
+  std::string file_path =
+      Path::JoinPath({Path::GetExecutablePath(), "testdata", "hello_world_elf_with_debug_info"});
+
+  auto hello_world = ElfFile::Create(file_path);
+  ASSERT_TRUE(hello_world) << hello_world.error().message();
```

Comment:
Didn't we want to avoid implicit conversions to bool?

Reply:
If so, please also change the usages below. Otherwise, feel free to ignore.

Reply:
I think the discussion was only about pointer and implicit conversions to bool. `hello_world` here is of type `ErrorMessageOr` and this is based on `outcome::result`.